devnet-sdk/shell/env/devnet.go

```diff
@@ -138,8 +139,12 @@ func fixupDevnetConfig(config *descriptors.DevnetEnvironment) error {
 		return fmt.Errorf("invalid L1 ID: %s", config.L1.ID)
 	}
 	if config.L1.Config == nil {
-		config.L1.Config = &params.ChainConfig{
-			ChainID: l1ID,
+		if l1Config := eth.L1ChainConfigByChainID(eth.ChainIDFromBig(l1ID)); l1Config != nil {
+			config.L1.Config = l1Config
+		} else {
+			config.L1.Config = &params.ChainConfig{
+				ChainID: l1ID,
+			}
```

Comment:
Do we want to proceed in this way without a full L1 Chain Config? Perhaps we should just error if we don't have one?

Reply:
It's a good point. However, this was how it worked before, so I'm content to leave it as-is for now to avoid breaking other code.